smart-contracts/erc20/assembly/__tests__/erc20.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { u256 } from "as-bignum/assembly";
+import { Exportable, ERC20 } from "..";
+import { Args } from "@massalabs/as-types";
+import { Address } from "@massalabs/massa-as-sdk";
+
+const totalSupply = u256.fromU32(1000);
+
+// @eslint-ignore @typescript-eslint/no-unused-vars
+// @ts-ignore
+@lazy @global const FungibleToken = new ERC20(totalSupply);
```

Comment:
I don't find the doc for `@lazy` neither `@global`. It's not here https://www.assemblyscript.org/concepts.html#code-annotations

Reply:
awesome!

Reply:
it open many more unknown possibilities! 🤩